crates/bevy_utils/src/label.rs

```diff
@@ -60,6 +60,47 @@ where
     }
 }
 
+/// Macro to define a new label trait
+///
+/// # Example
+///
+/// ```
+/// # use bevy_utils::define_old_style_label;
+/// define_old_style_label!(MyNewLabelTrait);
+/// ```
+#[macro_export]
+macro_rules! define_old_style_label {
```

Comment:
old_style_label feels like a very unclear name here

Reply:
Oh, I thought I had changed all of these. Must have missed one.